app/apps/client/admin/appManage.js

```diff
@@ -22,6 +22,10 @@ function getApps(instance) {
 
 	const appInfo = { remote: undefined, local: undefined };
 	return APIClient.get(`apps/${ id }?marketplace=true&version=${ FlowRouter.getQueryParam('version') }`)
+		.catch((e) => {
+			console.log(e);
```

Comment:
Whoops, I forgot about this. It can be removed since this error usually happens when they are viewing an app that is only installed locally.